package.json

```diff
@@ -9,14 +9,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "firebase": "^9.21.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-native": "^0.71.8",
```

Comment:
We never use `react-native` in this app!! Please do a code review by yourself before pushing the code.

Reply:
will delete